practices/managing-node-projects.md

```diff
@@ -17,25 +17,34 @@ The following guide describes how we use node and manage package dependencies in
 
 ## Specifying versions of node
 
-It is important to specify which versions of node your application expects. There are two ways of doing this, and you are encouraged to do both.
+It's important to specify which versions of node your application expects. There are two ways of doing this, and you are encouraged to do both.
 
 1. The [`engines`](https://docs.npmjs.com/files/package.json#engines) field in `package.json`:
-    * Important if authoring libraries, because `npm install` only _enforces_ compatibility on the basis of the `engines` field if the package is a dependency.
+    * `engines` is important if authoring libraries. Let's assume we are using a different version of node to that specified in `engines` field for package `foo`.
```

Comment:
👍 much clearer

>assume we are using

contractions lol: "we're using"

Reply:
you're giving me contractions